package.json

```diff
@@ -121,7 +122,7 @@
   "lint-staged": {
     "*.@(ts|tsx)": [
       "tslint --fix",
-      "yarn prettier-write --",
+      "npm run prettier-write --",
```

Comment:
I'd been having global prettier issues, probably due to my work when I first started using it for dev (so we could test the typescript implementation) seems fine after this